src/with-paginator-helper.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+/**
+ * Calls the given operation method with the parameters and if the autopagination
+ * argument is `true` it will do calls to the operation method until the LastEvaluatedKey value from the
+ * method response has a falsy value.
+ *
+ * @param {Function} methodFn Any DynamoDB operation method to call.
+ * @param {Object} params Parameters as expected by DynamoDB operation to use.
+ * @param {boolean} autopagination Whether to return all the records from DynamoDB or just the first batch of records.
+ * @returns {Promise} A promise that resolves to the response from DynamoDB.
+ */
+async function withPaginatorHelper(methodFn, params, autopagination) {
```

Comment:
I'd turn this into a factory function expecting an `operationFn` and returning another function that takes in _only_ `input` as argument ("operation" and "input" as that's what the `AWS.DynamoDB` typings typically calls them). The `autopagination` bit seems redundant to me.

```js
function withPaginator(operationFn) {
  async function autopaginate(input) { /* ... */ }
  return autopaginate;
}
```